### ISSUE TITLE
List formatting on concessions and contracts

### ISSUE THREAD

**Issue by caprenter** [NRGI/resourceprojects.org-frontend#265]
I can see consession and contract info here:
http://lodspeakr-live.nrgi-dev2.default.opendataservices.uk0.bigv.io/project/PE/anta-jciz8a.html

The list styling is a little odd to my tastes.

1) Does this have to be a list?

Does removing the margin and the bullet point help i.e.
    
margin-left: 0;
    padding-left: 0px;
    list-style-type: none;

**Comment by caprenter** [comment]
This is a separate issue, but I think can be bundled in here.

The query for a contract on a project page: https://github.com/NRGI/resourceprojects.org-frontend/blob/master/components/types/local_def__Project/queries/contracts.query

has 'title' as optional - but that is the bit that we try to display. If there is no title we get a bullet point but no text.

We could make the query require a title. Or we could check that e.g. we have data but no title and display that to the user.

The problem is seen here:
http://lodspeakr-live.nrgi-dev2.default.opendataservices.uk0.bigv.io/project/PH/apma-l9nepr.html

**Comment by timgdavies** [comment]
This does not need to be a list.

We shouldn't get data with a URL but not title, but we might.

Perhaps if we have no title, but we do have a URL we can just display 'Link' as the text?

**Comment by caprenter** [comment]
I can do this.

You might also like to check why the project listed above does have a contract but no title.
The Sheet with the data in does seem to have a title:
https://docs.google.com/spreadsheets/d/1S3aECv4WlUG1d0rIGVY7GJOdVNwWwPHR6z-r3J5Xnz0/edit#gid=279124923

**Comment by timgdavies** [comment]
Ah - it seems to have neither title nor URL. Looks like a model issue. Looking at it now...

**Comment by timgdavies** [comment]
Yes - we'd lost two properties from the model. I've just [updated the ontology here](https://github.com/NRGI/resource-projects-etl/commit/e51075744f118e9ba3e8b1f2084cc3d57f32e5c8) which, when deployed, and with PH data reloaded, should feed through the titles and URLs properly. 

I can manually load a fixed PH file if that would be useful.

**Comment by caprenter** [comment]
Cool - so I think that all we need to do here is style the list. It's no problem keeping it in, in case there is more than one record to display. If so it should look ok.

I think  a fixed file loaded up would be good. I've also just added that file to the fixtures for tests so I'll want to update that as well

**Comment by timgdavies** [comment]
Just regenerating fixed file now and will upload and share shortly.

**Comment by timgdavies** [comment]
Manually converted data [now loaded from here](http://etl.nrgi-dev2.default.opendataservices.uk0.bigv.io/dataload/20/)

**Comment by caprenter** [comment]
When I click the outputted TTL file on that page I get an error.

However I can see here: http://lodspeakr-live.nrgi-dev2.default.opendataservices.uk0.bigv.io/project/PH/apma-l9nepr.html
That the contract data is now showing.

Is there another way to send me a copy of the data pls?

**Comment by timgdavies** [comment]
Ben has updated the ontology on dev, so the conversion at http://etl.nrgi-dev2.default.opendataservices.uk0.bigv.io/dataload/15/ should now include the right triples and be available for download.

**Comment by caprenter** [comment]
Great, so I've got the data, I can confirm this is fixed locally. The tests also run with the new data - so all good.

I'll write an additional test to pick up the data here, apply the css fix and offer a commit soon.
